hack/test/internal/pkg/constants/constants.go

```diff
@@ -0,0 +1,10 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package constants
+
+const KubeVersion = "1.16.2"
+const KubeImage = "k8s.gcr.io/hyperkube:v" + KubeVersion
```

Comment:
I think we should source these from Talos constants. One source of truth for these values will be nice to have. Today when we need to update Kubernetes we have a number of different places we need to change. I'd like to get to a point where we only need to change 1 variable.

Reply:
Fixed in amended commit.